docs/GettingStarted.md

```diff
@@ -175,6 +175,47 @@ Net::HTTP module.
 
     content = Net::HTTP.get(URI('http://127.0.0.1/index.html'))
 
+### Sidekiq
+
+The Sidekiq integration is a server-side middleware which will trace job
+executions. It can be added as any other Sidekiq middleware:
+
+    require 'sidekiq'
+    require 'ddtrace'
+    require 'ddtrace/contrib/sidekiq/tracer'
+
+    Sidekiq.configure_server do |config|
+      config.server_middleware do |chain|
+        chain.add(Datadog::Contrib::Sidekiq::Tracer, debug: true)
```

Comment:
Apparently the last added middlewares are executed first, and `add()` append at the end.